curriculum/challenges/english/25-front-end-development/lab-fortune-teller/66c06d618d075c7f7f1b890a.md

```diff
@@ -0,0 +1,153 @@
+- `"You will find a new hobby soon."`.
+- `"It would be wise to avoid the colour red today."`.
+  
+1. You should select a random number between 1 and 5, inclusive, and assign it to the variable `randomNumber`.
+You should use an `if/else if` statement to print a different fortune based on the value of `randomNumber`:
+
+- If `randomNumber` is 1, print the first fortune.
+- If `randomNumber` is 2, print the second fortune.
+- If `randomNumber` is 3, print the third fortune.
+- If `randomNumber` is 4, print the fourth fortune.
+- If `randomNumber` is 5, print the fifth fortune.
+
+1. You should log the fortune to the console.
+   
+# --hints--
+
+You should initialize `fortune1` with a string value.
+
+```js
+assert.isNotNull(fortune1);
+assert.isString(fortune1);
+```
+
+You should initialize `fortune2` with the a string value.
+
+```js
+assert.isNotNull(fortune2);
+assert.isString(fortune2);
+```
+
+You should initialize `fortune3` with a string value.
+
+```js
+assert.isNotNull(fortune3);
+assert.isString(fortune3);
+```
+
+You should initialize `fortune4` with a string value.
+
+```js
+assert.isNotNull(fortune4);
+assert.isString(fortune4);
+```
+
+You should initialize `fortune5` with a string value.
+
+```js
+assert.isNotNull(fortune5);
+assert.isString(fortune5);
+```
+
+You should generate a random number between 1 and 5, inclusive, and assign it to the variable `randomNumber`.
+
+```js
+assert.isAtLeast(randomNumber, 1);
+assert.isAtMost(randomNumber, 5);
+```
+
+You should have a `selectedFortune` variable that is assigned a value based on the value of `randomNumber`.
+
+```js
+assert.isNotNull(selectedFortune);
+```
+
+You should use an `if` statement to check if `randomNumber` is equal to `1` and assign `selectedFortune` to `fortune1`.
+
+```js
+assert.match(__helpers.removeJSComments(code), /if\s*\(randomNumber\s*===\s*1\)\s*{\s*selectedFortune\s*=\s*fortune1;\s*}/);
+```
+
+You should create another condition using `else if` to check if `randomNumber` is equal to `2` and assign `selectedFortune` to `fortune2`.
+
+```js
+assert.match(__helpers.removeJSComments(code), /else\s*if\s*\(randomNumber\s*===\s*2\)\s*{\s*selectedFortune\s*=\s*fortune2;\s*}/);
+```
+
+You should create another condition using `else if` to check if `randomNumber` is equal to `3` and assign `selectedFortune` to `fortune3`.
+
+```js
+assert.match(__helpers.removeJSComments(code), /else\s*if\s*\(randomNumber\s*===\s*3\)\s*{\s*selectedFortune\s*=\s*fortune3;\s*}/);
+```
+
+You should create another condition using `else if` to check if `randomNumber` is equal to `4` and assign `selectedFortune` to `fortune4`.
+
+```js
+assert.match(__helpers.removeJSComments(code), /else\s*if\s*\(randomNumber\s*===\s*4\)\s*{\s*selectedFortune\s*=\s*fortune4;\s*}/);
+```
+
+You should create a final condition using `else` to assign `selectedFortune` to `fortune5`.
+
+```js
+assert.match(__helpers.removeJSComments(code), /else\s*{\s*selectedFortune\s*=\s*fortune5;\s*}/);
+```
```

Comment:
If you remove the if/else user story, you can remove these tests.

Reply:
I have removed the `if/esle` story and these tests.